docs/content/usage/packages/cargo.en-us.md

```diff
@@ -89,6 +89,14 @@ cargo publish
 
 You cannot publish a package if a package of the same name and version already exists. You must delete the existing package first.
 
+The server responds with the following HTTP Status codes.
+
+| HTTP Status Code  | Meaning |
+| ----------------- | ------- |
+| `201 Created`     | The package has been published. |
+| `400 Bad Request` | The package name, version, distribution, component or architecture are invalid. |
```

Comment:
I don't know if we should add these sections to packages which do not use `curl` or similar for uploading. With the original client (`cargo` is this case) you don't see the HTTP status code.

And copy-pasting does not work here because for Cargo there are no `distribution`, `component` or `architecture` parameters.

Reply:
OK. I will look over the packages that do not use `curl` and rephrase the sentence. Thanks for your information.